### DISCUSSION TITLE
Kandinsky

### DISCUSSION THREAD

**Discussion by koinkoin-project** [Woolverine94/biniou#10] [General]
Hello,

Juste une petite question naïve provenant du dernier commit ; le update.sh ajoute près de 35go de fichiers pour le nouveau kandinsky-3, et le kandinsky-2-1 est abandonné (ou ne fonctionne plus) ? Des modeles anciens sont parfois neutralisés/supprimés lors de certaines mises à jour de biniou ?

**Comment by Woolverine94** [comment]
Bonjour,

Les 35 Go me semblent correspondre à la taille du nouveau modèle Kandinsky 3.0. 

Le update.sh n'ajoute rien par lui-même, le modèle est téléchargé à la 1ère utilisation du module lorsque celui-ci est sélectionné. 

Kandinsky 2.1 n'est pas abandonné et devrait être fonctionnel.

Des modèles sont susceptibles d'être enlevés si ceux-ci sont obsolètes (quelques modèles peu performants et techniquement dépassés ont d'ailleurs été supprimés du chatbot cette semaine) ou si une révision plus récente du modèle est disponible. 

Cdt,

**Reply by koinkoin-project** [reply]
Autant pour moi, le fait d'avoir vu apparaître la version 3 m'a laissé penser que la 2-1 était abandonné mais encore listée (pour un eventuel nottoyage plus tard).

J'aurai dû (re)tester la 2-1 avant dslé jvais parfois un peu vite.

Merci pour ces réponses,
